nautobot/extras/utils.py

```diff
@@ -648,3 +650,62 @@ def bulk_delete_with_bulk_change_logging(qs, batch_size=1000):
         finally:
             change_context.defer_object_changes = False
             change_context.reset_deferred_object_changes()
+
+
+def create_schedule(schedule_data, job_kwargs, job_model, user, approval_required, task_queue=None):
```

Comment:
Can we make this signature a closer match to the `enqueue_job` signature, and get rid of the `schedule_data` parameters dict? Something more like:

```py
def create_schedule(
    job_model,
    user,
    *job_args,
    name=None,
    interval=JobExecutionType.TYPE_IMMEDIATELY,
    start_time=None,
    crontab=None,
    approval_required=True,
    task_queue=None,
    **job_kwargs,
):
```